src/components/layouts/pages/inspect/validationErrorMessages_en.json

```diff
@@ -115,11 +123,19 @@
       },
       "20": {
         "SUMMARY": "Is this a figure caption?",
```

Comment:
maybe better add "or table"

Reply:
Thanks for reviewing and for identifying this bug. I fixed it in `patch-19`.